src/main/java/de/theit/jenkins/crowd/CrowdConfigurationService.java

```diff
@@ -330,9 +353,12 @@ public Collection<GrantedAuthority> getAuthoritiesForUser(String username) {
         // Load the entry from cache if it's valid return it
         Collection<GrantedAuthority> authorities = getValidValueFromCache(username, authoritiesForUserCache);
         if (authorities != null) {
+            LOG.log(Level.FINEST, "getAuthoritiesForUser() cache hit: {0}", username);
             return authorities;
         }
 
+        LOG.log(Level.FINEST, "getAuthoritiesForUser() cache MISS: {0}", username);
+
```

Comment:
Please change this log to debug. Mostly because it could be pretty extensive in bigger organization.

Reply:
`FINEST` is the lowest predefined `Level` in `java.util.logging`, what do you suggest I change it to?

Reply:
oh... my bad. So leave is as it is